### PR TITLE
Add some API docs on how to retrieve the data

### LINKED ISSUE #88: Explain how to get the data back from a JSON request in the docs.
Maybe we should add a link to a page where questions are asked / answered (a FAQ, but for the users) where this solution could be explained.

### PR DESCRIPTION
**Pull request by almet**
Fix #88

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,50 @@ $ npm install
...
+
+One interesting property of the *userToken* is that it is actually half of the
+admin token !
+
+With that in mind, let's say we've generated a form with an *adminToken* of `152e3b0af1e14cb186894980ecac95de`. The *userToken* is then `152e3b0af1e14cb1`.
+
+So if we want to have access to the data on the server, using curl, we need to authenticate as the admin (using [BasicAuth](https://en.wikipedia.org/wiki/Basic_access_authentication) with `form:{adminToken}`):
+
+$ curl http://localhost:8888/v1/buckets/formbuilder/collections/152e3b0af1e14cb1/records \
+  -u form:152e3b0af1e14cb186894980ecac95de | python -m json.tool
```
Comment (leplatrem) [inline README.md]: nit: you could add intermediary variables to help reading

```
$ SERVER_URL="http://localhost:8888/v1"
$ ADMIN_TOKEN="152e3b0af1e14cb186894980ecac95de"
$ FORM_ID="152e3b0af1e14cb1"
$ curl $SERVER_URL/buckets/formbuilder/collections/$FORM_ID/records \
+  -u form:$ADMIN_TOKEN | python -m json.tool
```

File: README.md [hunk 2]
```diff
@@ -17,6 +17,50 @@ $ npm install
...
+- the *userToken*, that's used by users to find back the proper form.
+
+One interesting property of the *userToken* is that it is actually half of the
+admin token !
+
+With that in mind, let's say we've generated a form with an *adminToken* of `152e3b0af1e14cb186894980ecac95de`. The *userToken* is then `152e3b0af1e14cb1`.
+
+So if we want to have access to the data on the server, using curl, we need to authenticate as the admin (using [BasicAuth](https://en.wikipedia.org/wiki/Basic_access_authentication) with `form:{adminToken}`):
+
+$ curl http://localhost:8888/v1/buckets/formbuilder/collections/152e3b0af1e14cb1/records \
```
Comment (DirtyF) [inline README.md]: you forgot the ``` before the curl command, rendering is broken on GIthub, except that, it's pretty clear.

**Comment by almet** [comment]
Updated.

**Pull request merged**